rotkehlchen/data/importer.py

```diff
@@ -934,3 +934,87 @@ def import_nexo_csv(self, filepath: Path) -> Tuple[bool, str]:
                 except KeyError as e:
                     return False, str(e)
         return True, ''
+
+    def _consume_shapeshift_trade(self, csv_row: Dict[str, Any]) -> None:
+        """
+        Consume the file containing only trades from ShapeShift.
+        """
+        timestamp = deserialize_timestamp_from_date(
+            date=csv_row['timestamp'],
+            formatstr='iso8601',
+            location='ShapeShift',
+        )
+        buy_asset = symbol_to_asset_or_token(csv_row['outputCurrency'])
+        buy_amount = deserialize_asset_amount(csv_row['outputAmount'])
+        sold_asset = symbol_to_asset_or_token(csv_row['inputCurrency'])
+        sold_amount = deserialize_asset_amount(csv_row['inputAmount'])
+        rate = deserialize_asset_amount(csv_row['rate'])
+        fee = deserialize_fee(csv_row['minerFee'])
+        in_addr = csv_row['inputAddress']
+        out_addr = csv_row['outputAddress']
+        # TODO: determine exact time ShapeShift started using SAI with trades in csv exports
```

Comment:
Is this TODO still needed? you have a timestamp here. If this is indeed the right one, can you move it to the module level? There is no need to recompute it every time `_consume_shapeshift_trade()` is called.

Reply:
This is the date the MKR DAO launched MCD, so anything before that must have been SAI.  It is possible ShapeShift platform still generated CSVs with DAI until their systems were updated to reflect SAI sometime near this date.

I will move this to the module level and remove the TODO.

Reply:
You should also add it as a timestamp directly. No need to run `deserialize_timestamp_from_date()` since we know the timestamp and it won't change.

Reply:
@LefterisJP If I understand correctly, this would belong at `rotkehlchen/data/importer.py:1002` before the for loop and should be passed as a parameter to _consume_shapeshift_trade with the Timestamp value.

Is that correct?

Reply:
no no no. You should just add it at the top of the module:

```python
SAI_TIMESTAMP_OR_WHATEVER_NAME_THE_VAR_WILL_HAVE = 1574035200
```

somewhere here: https://github.com/rotki/rotki/blob/c4be84e692aa711a1cc7272d33d3209622ec40a0/rotkehlchen/data/importer.py#L33